### PR TITLE
Ensure the banner array always defines a width and a height

### PR DESCRIPTION
**Pull request by jeerbl**
Related to issue #1028

### CODE REVIEW

File: lib/max/Delivery/adSelect.php [hunk 1]
```diff
@@ -238,8 +238,8 @@ function MAX_adSelect($what, $campaignid = '', $target = '', $source = '', $with
             'bannerid'      => $row['bannerid'],
             'contenttype'   => $row['contenttype'],
             'alt'           => $row['alt'],
-            'width'         => $row['width'],
-            'height'        => $row['height'],
+            'width'         => (!isset($row['width']) || is_null($row['width'])) ? 0 : $row['width'],
+            'height'        => (!isset($row['height']) || is_null($row['height'])) ? 0 : $row['height'],
```
Comment (mbeccati) [inline lib/max/Delivery/adSelect.php]: Thanks for the PR. Simply `empty($row['height']) ? 0 : $row['height']` would be shorter here

**Comment by mbeccati** [comment]
Upon further inspection, the code you have modified is only invoked when a matching banner is found. Your report for #1028 was talking about a non-match instead. Which one is correct?

**Comment by jeerbl** [comment]
@mbeccati you are definitely right, my bad. I updated the pull request to now check the `width` and `height` values at JavaScript generation time in `afr.php` and rebuilt the `delivery` directory.

**Comment by mbeccati** [comment]
Merged manually in 66fd821ad

**Comment by mbeccati** [comment]
Thanks again!

**Pull request closed without merging**